test/perfverse/newrelic.yml

```diff
@@ -0,0 +1,15 @@
+common: &default_settings
+  app_name: Ruby Agent Perf testing
+  log_level: info
+  log_file_path: STDOUT
+  code_level_metrics:
+    enabled: true
```

Comment:
isn't this on by default?
```suggestion
```

Reply:
oh yeah I just copied this from the agent 2 query app, and didn't really think twice about it tbh